index.html

```diff
@@ -54,6 +54,197 @@ <h1 class="head">I’m Mariane<br>Glad to see<br> you!</h1>
+                </li>
+                <li>
+                    <h2 class="year">2015</h2>
+                </li>
+              </ul>
+            </div>
+            <p class="p-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="lang">
+                <a class="p-html" href="">html</a>
+                <a class="p-css" href="">css</a>
+                <a class="p-javas" href="">javaScript</a>
+            </div>
+            <div class="p-button">
+                <a class="p-proj" href="">See Project</a>
+            </div>
+           
+        </section>
+        <section class="card1">
+            <section class="portfolio-image3"></section>
+            <div class="p-layout">
+              <h1 class="portfolio-head">Tonic</h1>
+              <ul class="frame">
+                  <li>
+                      <h2 class="canopy">CANOPY</h2>
+                  </li>
+                  <li>
+                      <img src="Counter.png" alt="">
+                  </li>
+                  <li>
+                    <h2 class="backend">Back End Dev</h2>
+                  </li>
+                  <li>
+                    <img src="Counter.png" alt="">
+                </li>
+                <li>
+                    <h2 class="year">2015</h2>
+                </li>
+              </ul>
+            </div>
+            <p class="p-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="lang">
+                <a class="p-html" href="">html</a>
+                <a class="p-css" href="">css</a>
+                <a class="p-javas" href="">javaScript</a>
+            </div>
+            <div class="p-button">
+                <a class="p-proj" href="">See Project</a>
+            </div>
+           
+        </section>
+        <section class="card2">
+            <section class="portfolio-image4"></section>
+            <div class="p-layout">
+              <h1 class="portfolio-head">Tonic</h1>
+              <ul class="frame">
+                  <li>
+                      <h2 class="canopy">CANOPY</h2>
+                  </li>
+                  <li>
+                      <img src="Counter.png" alt="">
+                  </li>
+                  <li>
+                    <h2 class="backend">Back End Dev</h2>
+                  </li>
+                  <li>
+                    <img src="Counter.png" alt="">
+                </li>
+                <li>
+                    <h2 class="year">2015</h2>
+                </li>
+              </ul>
+            </div>
+            <p class="p-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="lang">
+                <a class="p-html" href="">html</a>
+                <a class="p-css" href="">css</a>
+                <a class="p-javas" href="">javaScript</a>
```

Comment:
- [ ] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`. Please kindly fix that.